extension/src/paymentHandler/payment-handler.js

```diff
@@ -122,21 +119,21 @@ function _validatePaymentRequest(paymentObject, authToken) {
     if (paymentValidator.hasErrors())
       return {
         success: false,
-        data: paymentValidator.buildCtpErrorResponse(),
+        errors: paymentValidator.buildCtpErrorResponse().errors,
       }
   } else {
     paymentValidator.validateMetadataFields()
     if (paymentValidator.hasErrors())
       return {
         success: false,
```

Comment:
I wonder if this flag is required at all? I guess if `errors` is not empty then it has the same meaning as above, right?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/656/commits/04d7a6bdceb15f77466d9008ff7b8dcfba7f2da1